### ISSUE TITLE
sv_cheats 1

### ISSUE THREAD

**Issue by ExterANAL** [danielkrupinski/OneByteMoney#1]
Is it possible to hack sv_cheats 1 to execute commands just in the console?

**Comment by taxxed** [comment]
Yes it is. Pretty self explanatory if you look at the code of his other onebyte/.py projects that edit a console cvar.
